posthog/api/dead_letter_queue.py

```diff
@@ -93,7 +88,6 @@ def get_queryset(self):
             fn_result = metric_context["fn"](0)  # type: ignore
             metric = {
                 "key": key,
-                "value": metric_context.get("value"),
```

Comment:
are you saying here we need to send value always? if we do, we should set it to None explicitly